src/ElementalChat.vue

```diff
@@ -22,7 +22,7 @@
               icon
               v-bind="attrs"
               v-on="on"
-              @click="editHandle()"
+              @click="needsHandle()"
```

Comment:
Isn't needsHandle a bool?

Reply:
horribly, there is a `needsHandle` bool and a `needsHandle` mutation. I'm gonna fix that so it's easier to read